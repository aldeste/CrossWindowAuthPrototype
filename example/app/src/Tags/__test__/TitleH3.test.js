import TitleH3 from "../TitleH3";
import renderer from "react-test-renderer";
import React from "react";
import "jest-styled-components";

describe("TitleH3 tag", () => {
  it("Should be defined", () => {
    expect(TitleH3).toBeDefined();
  });

  it("Matches earlier configuration", () => {
    const tree = renderer.create(<TitleH3 />).toJSON();
    expect(tree).toMatchStyledComponentsSnapshot();
  });
});
