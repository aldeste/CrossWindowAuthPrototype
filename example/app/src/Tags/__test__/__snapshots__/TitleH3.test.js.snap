// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleH3 tag Matches earlier configuration 1`] = `
.ipbgyT {
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
  font-size: 2rem;
  line-height: 1.09375;
  font-weight: 500;
  letter-spacing: 0.011em;
}

<h1
  className="TitleH3__Title-dYKMyV ipbgyT"
/>
`;
